SerialPrograms/Source/NintendoSwitch/Controllers/Joycon/NintendoSwitch_JoyconTable.cpp

```diff
@@ -87,11 +95,19 @@ JoyconStateRow::JoyconStateRow(EditableTableOption& parent_table)
     )
     , JOYSTICK_X(LockMode::UNLOCK_WHILE_RUNNING, 128, 0, 255)
     , JOYSTICK_Y(LockMode::UNLOCK_WHILE_RUNNING, 128, 0, 255)
+    , ACTION(false, LockMode::UNLOCK_WHILE_RUNNING, "", "")
 {
     PA_ADD_OPTION(DURATION);
     PA_ADD_OPTION(BUTTONS);
     PA_ADD_OPTION(JOYSTICK_X);
     PA_ADD_OPTION(JOYSTICK_Y);
+    PA_ADD_OPTION(ACTION);

```

Comment:
I would actually put this between `DURATION` and `BUTTONS`. IOW, put it near the beginning.